src/main/java/org/opensearch/sdk/SDKTransportService.java

```diff
@@ -107,6 +127,195 @@ public RemoteExtensionActionResponse sendRemoteExtensionActionRequest(RemoteExte
         );
     }
 
+    /**
+     * Requests that OpenSearch register the REST Actions for this extension.
+     *
+     * @param extensionRestPathRegistry The routes and classes which handle the REST requests
+     */
+    public void sendRegisterRestActionsRequest(ExtensionRestPathRegistry extensionRestPathRegistry) {
+        List<String> extensionRestPaths = extensionRestPathRegistry.getRegisteredPaths();
+        List<String> extensionDeprecatedRestPaths = extensionRestPathRegistry.getRegisteredDeprecatedPaths();
+        logger.info(
+            "Sending Register REST Actions request to OpenSearch for "
+                + extensionRestPaths
+                + " and deprecated paths "
+                + extensionDeprecatedRestPaths
+        );
+        AcknowledgedResponseHandler registerActionsResponseHandler = new AcknowledgedResponseHandler();
+        try {
+            transportService.sendRequest(
+                opensearchNode,
+                ExtensionsManager.REQUEST_EXTENSION_REGISTER_REST_ACTIONS,
+                new RegisterRestActionsRequest(uniqueId, extensionRestPaths, extensionDeprecatedRestPaths),
+                registerActionsResponseHandler
+            );
+        } catch (Exception e) {
+            logger.info("Failed to send Register REST Actions request to OpenSearch", e);
+        }
+    }
+
+    /**
+     * Requests that OpenSearch register the custom settings for this extension.
+     *
+     * @param customSettings A list of custom settings
+     */
+    public void sendRegisterCustomSettingsRequest(List<Setting<?>> customSettings) {
+        logger.info("Sending Settings request to OpenSearch");
+        AcknowledgedResponseHandler registerCustomSettingsResponseHandler = new AcknowledgedResponseHandler();
+        try {
+            transportService.sendRequest(
+                opensearchNode,
+                ExtensionsManager.REQUEST_EXTENSION_REGISTER_CUSTOM_SETTINGS,
+                new RegisterCustomSettingsRequest(uniqueId, customSettings),
+                registerCustomSettingsResponseHandler
+            );
+        } catch (Exception e) {
+            logger.info("Failed to send Register Settings request to OpenSearch", e);
+        }
+    }
+
+    private void sendGenericRequestWithExceptionHandling(
+        ExtensionRequestProto.RequestType requestType,
+        String orchestratorNameString,
+        TransportResponseHandler<? extends TransportResponse> responseHandler
+    ) {
+        logger.info("Sending " + requestType + " request to OpenSearch");
+        try {
+            transportService.sendRequest(opensearchNode, orchestratorNameString, new ExtensionRequest(requestType), responseHandler);
+        } catch (Exception e) {
+            logger.info("Failed to send " + requestType + " request to OpenSearch", e);
```

Comment:
We should probably change these (a few others in this class) to `.error()` log level.